manifests/params.pp

```diff
@@ -119,11 +123,7 @@
   $plugin_version          = 'installed'
 
   # Enable listening on http
-  if $::osfamily == 'RedHat' and versioncmp($::operatingsystemmajrelease, '7') <= 0 and fact('ipaddress6') {
```

Comment:
This test now needs to be updated:
https://github.com/theforeman/puppet-foreman_proxy/blob/c57d7fc64f2fa5a506f8531e482b081491dc7ff0/spec/classes/foreman_proxy__spec.rb#L105-L133
I don't think it really adds any value anymore after this change so it can be dropped.